src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * To style the map
+ */
+
+function createMap() {
+
+	// Create a new StyledMapType object, passing it an array of styles,
+	// and the name to be displayed on the map type control.
+	var styledMapType = new google.maps.StyledMapType(
+			[
```

Comment:
The indentation here is super weird but on my intellj is indented with two spaces.

Reply:
I think it is because you are using tabs instead of spaces. Please set your intellij to auto replace tab with 2 spaces so that such discrepancy on display does not happen. Also, please re-indent the file with space.

Reply:
![image](https://user-images.githubusercontent.com/26247692/59145050-5fedae00-8a11-11e9-87b7-916d1728589d.png)
Hmm..but on my intellj i've already configured to 2 spaces though. This is what it looks like on my intellj.

Reply:
Yes, by configuring your Intellij with displaying tab with 2 spaces will display properly on your intellij. However, the character you used to indent is still tab. Hence, to ensure display consistency, you need to replace the tabs with real spaces and please set you Intellij to auto replace tab with space, too.

Reply:
oh, thanks for letting me know. This is what you meant right?
![image](https://user-images.githubusercontent.com/26247692/59154470-6168b600-8aa5-11e9-8a08-f0575f51d4e3.png)

Reply:
I am not able to see whether it is a space or a tab from your screenshot. For me I use sublimeText editor that displays differently for tab and spaces. Maybe when you push up the changes I can checkout your branch and see or help you convert tabs to spaces.

Reply:
oh right oops okkay, for intellj it shows tabs/spaces at the bottom right corner. At first I didn't know what you meant coz I thought they were the same thing if I set tabs = 2 spaces. But yea I'll push the changes in abit together with the other changes.